src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplateBuilder.java

```diff
@@ -98,6 +99,8 @@ public class PodTemplateBuilder {
     private static final String WORKSPACE_VOLUME_NAME = "workspace-volume";
     public static final Pattern FROM_DIRECTIVE = Pattern.compile("^FROM (.*)$");
 
+    public static final String LABEL_KUBERNETES_INSTANCE = "kubernetes.jenkins.io/instance";
```

Comment:
_instance_ seems needlessly obscure. Why not _controller_?

Also, we already save a `buildUrl` which should be enough to identify the controller?

Reply:
`buildUrl` is an annotation so I can't use it as a filter when listing pods.